<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>teacher</class>
 <widget class="QWidget" name="teacher">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>911</width>
    <height>529</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background: center url(/home/tarek/Documents/QT projects/Ensia/assets/teacher-background.jpg)</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>911</width>
     <height>531</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 500 13pt &quot;Ubuntu Mono&quot;;
color:black;</string>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="QWidget" name="Home">
    <attribute name="title">
     <string>Tab 1</string>
    </attribute>
    <widget class="QGroupBox" name="groupBox">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>90</y>
       <width>571</width>
       <height>301</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background:rgb(205, 171, 143);</string>
     </property>
     <property name="title">
      <string>informations</string>
     </property>
     <widget class="QLineEdit" name="lineEdit_teacher_fullname">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>60</y>
        <width>211</width>
        <height>25</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color:black;
background:rgb(181, 131, 90);
border-radius:5px;</string>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_teacher_email">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>100</y>
        <width>211</width>
        <height>25</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color:black;
background:rgb(181, 131, 90);
border-radius:5px;</string>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_teacher_phone">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>180</y>
        <width>211</width>
        <height>25</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color:black;
background:rgb(181, 131, 90);
border-radius:5px;</string>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_teacher_id">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>220</y>
        <width>211</width>
        <height>25</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color:black;
background:rgb(181, 131, 90);
border-radius:5px;</string>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>60</y>
        <width>91</width>
        <height>17</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>Full name</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>100</y>
        <width>66</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Email</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>140</y>
        <width>131</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Date of birth</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>180</y>
        <width>66</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>phone</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>220</y>
        <width>66</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>ID</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_teacher_date">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>140</y>
        <width>211</width>
        <height>25</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color:black;
background:rgb(181, 131, 90);
border-radius:5px;</string>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="dashboard">
    <attribute name="title">
     <string>Tab 2</string>
    </attribute>
    <widget class="QFrame" name="frame">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>50</y>
       <width>711</width>
       <height>401</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background:rgb(205, 171, 143);</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>170</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-radius:4px;</string>
      </property>
      <property name="text">
       <string>Midterm</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>220</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-radius:4px;</string>
      </property>
      <property name="text">
       <string>Final :</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>120</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-radius:4px;</string>
      </property>
      <property name="text">
       <string>CC :</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>70</y>
        <width>111</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">
border-radius:4px;</string>
      </property>
      <property name="text">
       <string>Student ID:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_teacher_id_add">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>70</y>
        <width>151</width>
        <height>25</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background:rgb(181, 131, 90);
border-radius:4px;</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_teacher_cc_add">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>120</y>
        <width>113</width>
        <height>25</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background:rgb(181, 131, 90);
border-radius:4px;</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_teacher_midterm_add">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>170</y>
        <width>113</width>
        <height>25</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background:rgb(181, 131, 90);
border-radius:4px;</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_teacher_final_add">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>220</y>
        <width>113</width>
        <height>25</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background:rgb(181, 131, 90);
border-radius:4px;</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_2">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>120</y>
        <width>41</width>
        <height>25</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background:rgb(181, 131, 90);
border-radius:4px;</string>
      </property>
      <property name="text">
       <string>/ 20</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_3">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>170</y>
        <width>41</width>
        <height>25</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background:rgb(181, 131, 90);
border-radius:4px;</string>
      </property>
      <property name="text">
       <string>/ 20</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_4">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>220</y>
        <width>41</width>
        <height>25</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background:rgb(181, 131, 90);
border-radius:4px;</string>
      </property>
      <property name="text">
       <string>/ 20</string>
      </property>
     </widget>
     <widget class="QToolButton" name="toolButton">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>300</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-radius:5px;
background-color: rgb(192, 191, 188);</string>
      </property>
      <property name="text">
       <string>submit</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_teacher_course_add">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>260</y>
        <width>151</width>
        <height>25</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background:rgb(181, 131, 90);
border-radius:4px;</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_11">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>260</y>
        <width>121</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-radius:4px;</string>
      </property>
      <property name="text">
       <string>course name:</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="update">
    <attribute name="title">
     <string>Page</string>
    </attribute>
    <widget class="QFrame" name="frame_2">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>60</y>
       <width>711</width>
       <height>401</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background:rgb(205, 171, 143);</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QLabel" name="label_12">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>200</y>
        <width>151</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-radius:4px;</string>
      </property>
      <property name="text">
       <string>New password:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_14">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>150</y>
        <width>141</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-radius:4px;</string>
      </property>
      <property name="text">
       <string>Old  password:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_15">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>100</y>
        <width>111</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">
border-radius:4px;</string>
      </property>
      <property name="text">
       <string>ID:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_teacher_id_password">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>100</y>
        <width>151</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background:rgb(181, 131, 90);
border-radius:4px;</string>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_teacher_oldpassword">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>150</y>
        <width>151</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background:rgb(181, 131, 90);
border-radius:4px;</string>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
     </widget>
     <widget class="QToolButton" name="toolButton_2">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>250</y>
        <width>161</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-radius:5px;
background-color: rgb(192, 191, 188);</string>
      </property>
      <property name="text">
       <string>change password</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_teacher_newpassword">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>200</y>
        <width>151</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background:rgb(181, 131, 90);
border-radius:4px;</string>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>340</y>
        <width>66</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Note: </string>
      </property>
     </widget>
     <widget class="QPlainTextEdit" name="plainTextEdit">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>360</y>
        <width>621</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 9pt &quot;System-ui&quot;;
border-color: rgb(205, 171, 143);</string>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
      <property name="plainText">
       <string>Good password contains digits uppercase, lowercase, characters, and length have to be be more than 8</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_wrong_pass">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>180</y>
        <width>161</width>
        <height>17</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color:red;
font: 300 8pt &quot;Ubuntu&quot;;</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
